parsers/mysql/mysql.go

```diff
@@ -117,7 +116,7 @@ func (p *Parser) Init(options interface{}) error {
 		url := fmt.Sprintf("%s:%s@tcp(%s)/", p.conf.User, p.conf.Pass, p.conf.Host)
 		db, err := sql.Open("mysql", url)
 		if err != nil {
-			log.Fatal(err)
+			logrus.Fatal(err)
 		}
 		// run one time queries
 		p.hostedOn = getHostedOn(db)
```

Comment:
I'm not sure why I didn't suggest this before - I really should have.  Given that the log messages are all the same, we should just return (*returnval, error) for each of the get* methods, and log the messages at the call sites instead of at every error check within them.